src/linux/backend-v4l2.cpp

```diff
@@ -841,7 +857,8 @@ namespace librealsense
             {
                 try
                 {
-                    if (!(cur_node.first.uvc_capabilities & V4L2_CAP_META_CAPTURE))
+                    /* mi: 3 - metadata node, IPU6 metadata is mplane type */
+                    if (! ((cur_node.first.uvc_capabilities & V4L2_CAP_META_CAPTURE) || (cur_node.first.mi == 3 && cur_node.first.uvc_capabilities & V4L2_CAP_VIDEO_CAPTURE_MPLANE)))
```

Comment:
hardcoded mi == 3 is specific to alder lake - so this will not be mergeable to main branches

Reply:
I agree it's not optimal, but currently the only kernel that report MPLANE is ADL-P.
It's an OR with the older condition.
Any other suggestions?

Reply:
maybe the pid can be checked before trying the mi 3, in order to minimize the risks - but I understood your answer too, and it is ok also without it (for now, since it is the only UC we have with multiplane)